apps/frontend/src/features/admin/components/ReportsContainer.tsx

```
@@ -31,79 +42,110 @@ export const ReportsContainer = ({ reports, loading, error }: ReportsContainerPr
-            유저
-          </div>
-          <div css={headerCellStyle(theme)} role="columnheader" aria-label="날짜" tabIndex={0}>
-            날짜
-          </div>
-        </div>
-        {reports.length === 0 ? (
-          <div css={emptyRowStyle(theme)} role="row">
-            신고가 없습니다.
+    <div css={containerStyle}>
+      <div css={tableWrapperStyle}>
+        <div css={gridStyle(theme)} role="table" aria-label="신고 목록">
```

Biome (2.3.14)

[error] The elements with this role can be changed to the following elements:
<table>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)